### DISCUSSION TITLE
Bandwidth Usage - top download/upload users

### DISCUSSION THREAD

**Discussion by IzacFrank** [Security-Onion-Solutions/securityonion#6596] [Unsupported Versions]
Dear,

In older version of SO, it was possible to monitor bandwidth usage (Kibana > Connections > Top Total Bytes Dashboard). Is it possible to monitor in the latest version? Could you please instruct me?

Thank you.
Isac

**Comment by dougburks** [comment]
If you look at how the dashboard was built in 16.04, you should be able to build something similar in the latest version.

**Reply by IzacFrank** [reply]
Thank you @dougburks  but I couldn't find anything related 16.04, either. Do you have any link or example link you can share?

**Reply by dougburks** [reply]
> I couldn't find anything related 16.04, either

I'm not sure I understand your response. Earlier you said:

> In older version of SO, it was possible to monitor bandwidth usage (Kibana > Connections > Top Total Bytes Dashboard)

So you should be able to look in Security Onion 16.04 at the Dashboard you are referring to, see how it was built, and then build it similarly in the latest version.
